src/swagger-specs/segmentation.yaml

```diff
@@ -3115,27 +3098,29 @@ components:
         creationTime:
           type: integer
           example: 1650374572000
-          description: The time when the audience was created in UTC.
+          description: The time when the audience, in milliseconds, was created in UTC.
```

Comment:
"A UTC timestamp for when the audience was created, in milliseconds." (Similar rephrasing for other lines, minimizing commas and having the time unit interrupting the meaning of the timestamp)